src/app/login/page.tsx

```diff
@@ -0,0 +1,63 @@
+'use client';
+
+import { FC, useState } from 'react';
+import Link from 'next/link';
+
+const LoginForm: FC = () => {
+  const [userData, setUserData] = useState({
+    email: '',
+    password: '',
+  });
+
+  return (
+    <section className="sec-cont login-container">
+      <div>
+        <h1>Inloggning</h1>
+        <form action="">
```

Comment:
Prevent default goes here.
Use this:
```tsx
<form
     action=""
     onSubmit={(e) => {
       e.preventDefault();
       console.log(userData);
     }}
></form>
```

Reply:
I think we can write the submit when we write the logic for submitting the form, in another PR.